lib/barbeque_client/executor.rb

```diff
@@ -8,9 +8,19 @@ class Executor
     # @param [String] queue_name - barbeque's job_queues.name
     def initialize(job:, message:, message_id:, queue_name:)
       @job        = job
-      @message    = JSON.load(message)
```

Comment:
activejob gem has dependency of activesupport gem. Since this module depends on ActiveJob, you can never drop ActiveSupport dependency from this module. So generally I don't think it's worth removing ActiveSupport dependency <i>from Rails-related code</i> but it's up to you.